connectathon-scenarios-2019-09/examples/scenario-02-outbound-request.json

```diff
@@ -13,9 +14,16 @@
       "entry": [{
         "resource": {
           "resourceType": "ServiceRequest",
-          "id": "Example-MRI-Request",
+          "id": "example-MRI-59879846",
           "status": "draft",
           "intent": "plan",
+          "code": {
+            "coding": [{
+              "system": "http://www.ama-assn.org/go/cpt",
+              "code": "71275"
```

Comment:
Since this field was missing, I just chose one.  Let me know if there's a more appropriate value.

Reply:
Thanks -- works for now :)